Configuration/ProcessModifiers/python/seedingDeepCore_cff.py

```diff
@@ -0,0 +1,5 @@
+import FWCore.ParameterSet.Config as cms
+
+# This modifier is for activating DeepCore seeding for the JetCore tracking iteration
+
+seedingDeepCore =  cms.Modifier()
```

Comment:
the checker is suggesting to add an extra line, I guess

Reply:
done in the last commit